dpar-utils/tensorflow/write-graph

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+#  -*- coding: utf-8 -*-
+
+import argparse
+import sys
+import tensorflow as tf
+import toml
+
+from config import DefaultConfig
+from model import ParseModel
+
+
+def usage():
+    print("Usage: %s write-graph SHAPES GRAPH" % sys.argv[0])
+    sys.exit(1)
+
+
+def create_graph(config, shapes, graph_filename):
+    gpuopts = tf.GPUOptions(per_process_gpu_memory_fraction=0.3)
+    tfconfig = tf.ConfigProto(gpu_options=gpuopts)
+
+    with tf.Graph().as_default(), tf.Session(config=tfconfig) as session:
+        initializer = tf.random_uniform_initializer(-config.init_scale,
+                                                    config.init_scale)
+
+        with tf.variable_scope("model", reuse=None, initializer=initializer):
+            m = ParseModel(config=config, shapes=shapes)
+
+        init = tf.variables_initializer(tf.global_variables(), name='init')
+
+        saver = tf.train.Saver(tf.global_variables())
+
+        tf.train.write_graph(
+            session.graph_def,
+            './',
+            graph_filename,
+            as_text=False)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        'shape_file',
+        metavar='SHAPES_FILE',
```

Comment:
Could it lead to confusion if it's 'shape_file' in the line above and 'SHAPE**S**_FILE' here?

Reply:
Indeed, I'll change that.